### ISSUE TITLE
This is an instance of UUID

### ISSUE THREAD

**Issue by lahirug** [outworkers/phantom#447]
I get this message every bind I do and I think this is degrading the performance of my application drastically when insert couple of thousand inserts with batches. I can see this is removed but there is no release after that (I am using 1.21.5). I should be using 1.21.5 because of the requirements for batches with preparestatement.

Is there any help I can get on this ? Do we have a build without this where I can pull from sbt ?

**Comment by alexflav23** [comment]
Hi @lahirug, both `1.22.0` and `1.22.1` have been released long ago. You have both badges and changelogs on the repo to show you that.

**Comment by lahirug** [comment]
When I went to release page it shows me latest release is 1.21.5 with green label. Sorry, I will try with 1.22.1. Thanks.

**Comment by lahirug** [comment]
Even after moving to 1.22.1 I still see the Console out.

**Comment by alexflav23** [comment]
Hi @lahirug ,

Sorry to hear that, but I'll need a bit more information before I'm able to help you. I do remember a problem with prepared statements, but that's been resolved a very long time ago. Could you please provide a complete example?

So far all I have is "it doesn't work", and those kind of questions we address on a day rate.

Regards.

**Comment by psycho-ir** [comment]
@lahirug Your issue should be resolved by this PR:
https://github.com/outworkers/phantom/pull/431

From what I see 1.22.1 released 19 days ago. So this bugfix should not be a part of the latest release.

As an interim solution you can disable console print at all (I assume your application has nothing todo with stdout)

`
  //A fast solution to solve unwanted stdout writes from phantom project.
  Console.setOut(new OutputStream {
    override def write(b: Int): Unit = {}
  })
`

**Comment by alexflav23** [comment]
I now understand what you meant. We will do a new release today to include this.

**Comment by lahirug** [comment]
@alexflav23 Thanks I will will be keep looking at release page.
@psycho-ir where should I add this ? I tried adding it in the place where I create tables during startup, but doesn't compile.

```
missing arguments for method withOut in object Console;
[error] follow this method with `_' if you want to treat it as a partially applied function
[error]     Console.withOut(new OutputStream { override def write(b: Int): Unit = {} })
[error]                    ^
[error] one error found
[error] (fatline/compile:compileIncremental) Compilation failed
[error] Total time: 6 s, completed Mar 14, 2016 9:28:00 AM
```

**Comment by psycho-ir** [comment]
@lahirug You are using withOut! which is not appropriate here.
Try setOut (Deprecated but works as an interim solution)
Run that code in the first step of your application initialization.

**Comment by alexflav23** [comment]
@lahirug @psycho-ir Fixed in `1.23.0`, now out on Bintray.

**Comment by lahirug** [comment]
Hi @alexflav23, 

It solved my issue...Thanks for instant support.

Lahiru
